src/components/ReactFlagsSelect/ReactFlagsSelect.test.tsx

```diff
@@ -131,11 +155,29 @@ describe("ReactFlagsSelect", () => {
         expect(screen.getByText("US")).toBeInTheDocument();
       });
 
+      it("renders a custom label of each country option if present", () => {
```

Comment:
This is kind of a repetition of the previous test above it (same description too).
I think you can use the previous test and modify one label to use the CustomLabel object